components/operationModal/index.js

```diff
@@ -0,0 +1,26 @@
+import TransactionOperation from '@components/transaction-operation';
+import React, { useEffect, useState } from 'react';
+import receivers from '../../mock/receivers';
+const OperationModal = (props) => {
+  const { transactionType, personData } = props;
+  const [modal, showModal] = useState(false);
+  useEffect(() => {
+    console.log(transactionType);
```

Comment:
Please make sure to remove console statements

Reply:
Good catch sir